server/config/db.config.js

```diff
@@ -1,12 +1,19 @@
 import pg from 'pg';
 import dotenv from 'dotenv';
+import parseUrl from 'parse-database-url';
+
 import config from './config';
 
 dotenv.config();
 const { Pool } = pg;
+let configInfo;
```

Comment:
What if the NODE_ENV is test? configInfo will be undefined variable. You may want to take care of a situation where the environment is not known.